### DISCUSSION TITLE
Single login

### DISCUSSION THREAD

**Discussion by joseruizdev** [tenancy/tenancy#234] [Q&A]
Is it possible to use a single login for all the tenants without having to type the subdomain?

**Comment by sniper7kills** [comment]
Should be possible; Can you give some more info as to what exactly you are trying to do.

If all of the tenants use the same domain; but different subdomains; And your users are stored in System database (not on a tenant database) the following link show allow you to use authentication as it is out of the box: https://stackoverflow.com/questions/34488477/authuser-doesnt-travel-across-subdomains-laravel-5-2

You'll need to handle user access to tenants; but that should be doable though middleware.

**Reply by thumbtech** [reply]
FWIW if you're using cookie session driver you'll want to change the session domain setting to your "main" domain, e.g.,

Tenants in something like:

- tenant1.example.com
- tenant2.example.com
- etc.

The session domain setting should be ".example.com", easiest handled in .env with

`SESSION_DOMAIN=.example.com`

That way auth session will be maintained when redirecting in & out of main & sub domains.
